### PR TITLE
Convert PDF to PNG before running OCR

### PR DESCRIPTION
**Pull request by alexk307**
Adds the ability to convert PDF documents into PNG images of their first page. This is controlled by the `pdf_to_png` option.

image

### CODE REVIEW

File: configs/python/backend/backend.yaml [hunk 1]
```diff
@@ -327,10 +327,13 @@ scanners:
           - 'type_is_tiff'
           - 'image/x-ms-bmp'
           - 'bmp_file'
+          - 'application/pdf'
+          - 'pdf_file'
       priority: 5
       options:
         extract_text: False
         tmp_directory: '/dev/shm/'
+        pdf_to_png: True
```
Comment (rw-access) [inline configs/python/backend/backend.yaml:336]: IIRC you need to update `scanners.yaml` too?

Comment (alexk307) [reply configs/python/backend/backend.yaml:336]: I don't think I do. That file doesn't seem to get used in our build. I can make them the same though.

Comment (alexk307) [reply configs/python/backend/backend.yaml:336]: added

Comment (rw-access) [reply configs/python/backend/backend.yaml:336]: Looks like it's the other way around?
https://github.com/sublime-security/strelka/blob/439d1557422ed57d6885de97c6aa17e3eabfe869/build/python/backend/Dockerfile#L151

**Comment by alexk307** [comment]
@jkamdjou is this something that we'd want to PR back to the main Strelka repo?

**Review by rw-access — APPROVED** [review]

File: configs/python/backend/backend.yaml (hunk 1 above)
Comment (alexk307) [reply configs/python/backend/backend.yaml:336]: hmph - good catch

**Pull request merged**